segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeper/BookKeeperMetrics.java

```diff
@@ -68,7 +65,7 @@ void writeCompleted(Duration elapsed) {
 
         void bookKeeperWriteCompleted(int length, Duration elapsed) {
             this.writeLatency.reportSuccessEvent(elapsed);
-            this.writeBytes.reportSuccessValue(length);
+            DYNAMIC_LOGGER.incCounterValue(MetricsNames.BK_WRITE_BYTES, length);
```

Comment:
If you do this, will you still be able to find the BK write throughput? This was useful for that purpose.

Reply:
Yes, actually @kmgowda is able to easily do that with the current counter.